waltz-ng/client/data-types/routes.js

```diff
@@ -0,0 +1,42 @@
+const baseState = {
+    resolve: {
+        dataTypes: [
+            'DataTypesService',
+            (dataTypesService) => dataTypesService.loadDataTypes()
+        ]
+    }
+};
+
+
+const listState = {
+    url: 'data-types',
+    views: {'content@': require('./data-type-list') }
+};
+
+
+const viewByCodeState = {
+    url: 'data-types/code/{code}',
+    views: {'content@': require('./data-type-view') },
+    redirectTo: 'main.data-type.view',
```

Comment:
They can but I think we discussed a redirect to the Id version. That's what
I have in my notes. If you prefer the non redirect option I can do it that
way?

On 12 Aug 2016 15:42, "David Watkins" <notifications@github.com> wrote:

> In waltz-ng/client/data-types/routes.js
> <https://github.com/khartec/waltz/pull/417#discussion_r74602409>:
>
> > +            (dataTypesService) => dataTypesService.loadDataTypes()
> > +        ]
> > +    }
> > +};
> > +
> > +
> > +const listState = {
> > +    url: 'data-types',
> > +    views: {'content@': require('./data-type-list') }
> > +};
> > +
> > +
> > +const viewByCodeState = {
> > +    url: 'data-types/code/{code}',
> > +    views: {'content@': require('./data-type-view') },
> > +    redirectTo: 'main.data-type.view',
>
> why do we need a redirect ? Can't both routes point the same
> page/controller combo ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/khartec/waltz/pull/417/files/e4bc887bdadc42f03d5e44e0876776ce30ea1a5c#r74602409>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC4520tH5eq6t5ysPrrPEEGklNpNxNKCks5qfIZTgaJpZM4JiBOI>
> .
>